source/jobs/README.md

```diff
@@ -0,0 +1,232 @@
+"args": ["ifupdown"],
+...
+```
+If we wanted our remote device to print out "Hello World", then our job document so far might look like:
+```
+{
+    "operation": "echo",
+    "args": ["Hello world!"]
+}
+```
+
+`path` *string* (Optional): This field tells the Jobs feature where it should look to find an executable that matches the specified
+operation. If this field is omitted, the Jobs feature will assume that the executable can be found in its path. If
+you expect that the executable (operation) should be found in the Job feature's handler directory 
+(~/.aws-iot-device-client/jobs by default or specified via command line/json configuration values), then the path
+must be specified as part of the job document as follows:
+```
+...
+"path": "default",
+...
+```
+The path could also be specified as a particular directory that is expected on the device, such as:
+```
+...
+"path": "/home/ubuntu/my-job-handler-folder"
+...
+```
+
+`includeStdOut` *boolean* (Optional): This field tells the Jobs feature whether the standard output (STDOUT) of the child process
+should be included in the job execution update that is published to the IoT Jobs service. For example:
+```
+...
+"includeStdOut": true,
+...
+```
+
+`allowStdErr` *integer* (Optional): This field tells the Jobs feature whether a specific number of lines of output on STDERR
+from the job handler may be acceptable. If this field is omitted, the Jobs feature will use a default of 0 (zero), meaning
+that any lines of output on STDERR will cause the job to be marked as failed. For example: 
+```
+...
+"allowStdErr": 2,
+...
+```
+This job document indicates that the Jobs feature will mark the job as a success if the return code of the process is 0 AND
+there are fewer than or equal to 2 (two) lines of standard error output from the job handler. 
+
+### Creating your own Job Handler
+
+The Jobs feature within the AWS IoT Device Client handles establishing and maintaining a connection to AWS IoT Core and making
+sure that jobs are delivered to your device, but Job handlers represent the actual business logic that you want to execute on your
+device. 
+
+For example, we might want to create a new application written in Python called "foo" that should be run whenever our
+device receives a "foo" job. After writing the foo application, which for this particular example might only consist of a single
+file called "Foo.py", here's the steps that we would need to go through to set our AWS IoT Device Client to automatically trigger
+foo:
+
+1. Add Foo.py to our Job Handler Directory: By default, the Jobs feature will look at ~/.aws-iot-device-client/jobs/ for a handler, 
+so we should put Foo.py here (where `~/` represents the home directory of the user that will run the Device Client - /root if we're
+running the AWS IoT Device Client as a service, or /home/ubuntu/ if we're running under the default Ubuntu user). The new path
+then looks something like /home/ubuntu/.aws-iot-device-client/jobs/Foo.py. 
```

Comment:
nitpick: Let's put the paths, commands and file names between tildes so that they're formatted appropriately.